packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -631,6 +631,22 @@ export default class NewVm extends BaseComponent {
         )
     }
   )
+
+  _findNetwork = () => {
+    const automaticNetwork = find(
```

Comment:
If there are multiple "automatic" networks, I think we want to use them all.

Reply:
Hum, by adding many interface with each automatic network?

Reply:
Yes, according to the issue, it seems to be the wanted (XenCenter's) behavior.